src/psyclone/psyir/nodes/omp_task_directive.py

```diff
@@ -0,0 +1,2136 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2021-2022, Science and Technology Facilities Council.
```

Comment:
Copyright year please. In fact, should this be 2022-2023?

Reply:
I will do 2022-23, I don't think I started this file before 2022.